nexus-description.txt

```diff
@@ -1,9 +1,35 @@
 [center][b]ParallaxGen is in ALPHA. It has been successfully used by many but is still in development. Please be patient and file detailed bug reports if you encounter any.[/b][/center]
-
 [center]The GitHub page for this open-source project can be found [url=https://github.com/hakasapl/ParallaxGen]here[/url].[/center]
+[b][size=6][center][color=#6aa84f]Preconditions[/color][/center][/size][/b]
+[list]
+[*][size=3]Make sure [url=https://aka.ms/vs/17/release/vc_redist.x64.exe]Microsoft Visual C++ Redistributable x64[/url] is installed[/size]
+[*][size=3]Make sure .NET 8 runtime is installed, you can get it here [url=https://dotnet.microsoft.com/en-us/download/dotnet/8.0]Download .NET 8.0[/url][/size]
+[/list]
+[b][size=6][center][color=#6aa84f]Overview[/color][/center][/size][/b]
+[size=3]ParallaxGen scans the textures of your load order and creates patched meshes and a plugin to enable [url=https://en.wikipedia.org/wiki/Parallax_mapping]Parallax mapping[/url] and/or [url=https://modding.wiki/en/skyrim/developers/complex-parallax-materials]Complex Materials[/url]. You need this tool in the following cases:[/size]
+[list]
+[*][size=3]you installed a mod that contains parallax or complex material textures but does not contain parallax/complex material meshes, note that this is the recommended way to provide textures[/size]
+[*][size=3]you created your own parallax textures[/size]
+[*][size=3]you do have parallax/complex material meshes, but they are overridden by another mod that does not have parallax/complex material enabled[/size]
```

Comment:
Add a point here about the fact that you might have different shader types in the same mesh